data/plugins/language_python.lua

```diff
@@ -6,8 +6,12 @@ syntax.add {
   files = { "%.py$", "%.pyw$", "%.rpy$" },
   headers = "^#!.*[ /]python",
   comment = "#",
+  block_comment = '"""',
   patterns = {
     { pattern = { "#", "\n" },                 type = "comment"  },
+    { regex   = { '^\\s*"""', '"""' },         type = "comment"  },
+    { regex   = '[uUrR](?=")',                 type = "keyword"  },
```

Comment:
If you want to use a different token type for `uUrR`, you should change the other patterns for strings.

Also, this is doable with Lua patterns:
```suggestion
    { pattern = '[uUrR]%f["]',                 type = "keyword"  },
```